easybuild/main.py

```diff
@@ -615,7 +611,6 @@ def main(args=None, logfile=None, do_build=None, testing=False, modtool=None, pr
 
     options, orig_paths = eb_go.options, eb_go.args
 
-    global _log
```

Comment:
This one is needed isn't it as you set it below

Reply:
Oh right, that's kind of hidden...

You're right.